modules/quantcastIdSystem.js

```diff
@@ -34,9 +192,28 @@ export const quantcastIdSubmodule = {
    * @function
    * @returns {{id: {quantcastId: string} | undefined}}}
    */
-  getId() {
+  getId(config) {
     // Consent signals are currently checked on the server side.
     let fpa = storage.getCookie(QUANTCAST_FPA);
+
+    const coppa = coppaDataHandler.getCoppa();
+    if (coppa) {
```

Comment:
why do we have a separate piece of code here handling coppa checks in a different place to the GDPR and CCPA consent checks.  I think the behaviour can / should just be the same for all of these